convert2rhel/rhelvariant.py

```diff
@@ -35,6 +35,16 @@ def determine_rhel_variant():
         tool_opts.variant = _user_to_choose_rhel_variant()
     loggerinst.info("Variant: %s" % tool_opts.variant)
 
+    # checking RHN Classic
+    loggerinst.info("Checking RHN Classic")
+
+    from convert2rhel import subscription
+    if subscription.is_rhn_classic_used():
```

Comment:
```suggestion
    if subscription.is_rhn_classic_used():
```
Please move this condition to `unregister_from_rhn_classic()`.